src/main/java/com/fullstackweb_app/fullstackweb_app/AlbumService.java

```diff
@@ -0,0 +1,24 @@
+package com.fullstackweb_app.fullstackweb_app;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class AlbumService {

```

Comment:
This class isn't a pattern we've ever talked about using; I'm not sure where it came from.